user/user-guide.md

```diff
@@ -697,21 +705,20 @@ The search REST call returns a JSON object of the following form:
       }
   }
 }
-
 ```
 
 ### Additional notes
 
 #### Date format
 
-Dates and times must be specified int the ISO-8601 format
+Dates and times must be specified in [ISO-8601](https://www.iso.org/iso-8601-date-and-time-format.html) format.
```

Comment:
(non-blocking) Maybe more useful to link to the corresponding rfc https://tools.ietf.org/html/rfc3339

Reply:
Though technically more correct, reading the IETF link would take 10min, and reading this link takes 10s :)